src/ctapipe/reco/stereo_combination.py

```diff
@@ -74,23 +76,18 @@ class StereoMeanCombiner(StereoCombiner):
         ),
     ).tag(config=True)
 
-    log_target = Bool(
-        False,
-        help="If true, calculate exp(mean(log(values))).",
+    prefix = Unicode(
+        default_value="",
+        help="Prefix to be added to the output container / column names.",
     ).tag(config=True)
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-
-        self.supported = {
-            ReconstructionProperty.ENERGY,
-            ReconstructionProperty.GEOMETRY,
-            ReconstructionProperty.PARTICLE_TYPE,
-        }
-        if self.property not in self.supported:
-            raise NotImplementedError(
-                f"Combination of {self.property} not implemented in {self.__class__.__name__}"
-            )
+    property = UseEnum(
```

Comment:
Does it make sense to make this configurable? Shouldn't it just be `GEOMETRY`?

Reply:
In principle, yes - it should just be `GEOMETRY`, but the `StereoCombiner` is mainly instantiated via `.from_name()` in the `Reconstructor`-subclasses and checked by the `property`-trait to ensure that a valid `StereoCombiner` is selected for each `Reconstructor`, right? In that case, I definitely think it makes sense to prevent users from selecting the wrong `StereoCombiner` in the config and raising a proper error.
Moreover, even if `GEOMETRY` is the only reconstructable property, I don't think it's such a bad idea to require users to explicitly specify the property they want to reconstruct - even if they init the `StereoCombiner` directly, independently of a `Reconstructor`. It prevents misconfigurations early on.